CRM/Civicase/Upgrader/Steps/Step0011.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * CRM_Civicase_Upgrader_Steps_Step0011 class.
+ */
+class CRM_Civicase_Upgrader_Steps_Step0011 {
+
+  /**
+   * Performs Upgrade.
+   */
+  public function apply() {
+    $this->updateFileUploadCategoryGrouping();
+
+    return TRUE;
+  }
+
+  /**
+   * Update file upload category grouping.
+   */
+  private function updateFileUploadCategoryGrouping() {
+    try {
+      // Find category.
+      $category = civicrm_api3('OptionValue', 'getsingle', [
+        'name' => 'File Upload',
+        'component_id' => 'CiviCase',
+        'option_group_id' => 'activity_type',
```

Comment:
We can add one more parameter here 
```php
'grouping' => ['!=' => 'file']
```
That way an insert won't be done again if the option value has been updated to `file` before.
Not likely the upgrader will be ran twice but more like our standard for making upgraders idempotent.